src/themes/five/src/components/BusinessListingSearch/index.js

```diff
@@ -74,7 +74,7 @@ export const BusinessListingSearchUI = (props) => {
     handleUpdateProducts
   } = props
 
-  const [orderState] = useOrder()
+  const [{ carts }, orderState] = useOrder()
```

Comment:
You're using this hook the wrong way. please take a look to the context

Reply:
What is the better way to do that? or this is a backend issue